bloom-inference-server/benchmark.py

```diff
@@ -5,12 +5,18 @@
 
 import deepspeed
 import torch
-
 import utils
-from models import Model, get_model_class
-from utils import (BENCHMARK, DS_INFERENCE, DS_ZERO, GenerateRequest,
-                   get_argument_parser, get_dummy_batch, parse_generate_kwargs,
-                   print_rank_n, run_and_log_time)
+from models import Model
+from models import get_model_class
+from utils import BENCHMARK
+from utils import DS_INFERENCE
+from utils import DS_ZERO
+from utils import GenerateRequest
+from utils import get_argument_parser
+from utils import get_dummy_batch
+from utils import parse_generate_kwargs
+from utils import print_rank_n
+from utils import run_and_log_time
```

Comment:
this looks quite bad.

perhaps we adopt the same setup as `transformers`? If it resonates here is the setup to copy:

https://github.com/huggingface/transformers/blob/0d1ba2dd0ba1b584c0380ee9f69c2b222b462d76/Makefile#L68-L69

https://github.com/huggingface/transformers/blob/0d1ba2dd0ba1b584c0380ee9f69c2b222b462d76/setup.cfg#L1-L47

https://github.com/huggingface/transformers/blob/0d1ba2dd0ba1b584c0380ee9f69c2b222b462d76/pyproject.toml#L1-L3

Reply:
then yes, by all means you can expand it to the rest of the code.